ros_gz_sim_demos/launch/air_pressure.launch.xml

```diff
@@ -0,0 +1,13 @@
+<launch>
+  <gz_server
+    world_sdf_file="sensors.sdf"
+    use_composition="True"
+    create_own_container="True" />
+  <ros_gz_bridge
+    bridge_name="ros_gz_bridge"
+    config_file="$(find-pkg-share ros_gz_sim_demos)/config/air_pressure.yaml"
+    use_composition="True"
+    bridge_params="'qos_overrides./air_pressure.publisher.reliability': 'best_effort'" />
+  <node pkg="rqt_topic" exec="rqt_topic" />
```

Comment:
Required https://github.com/ros-visualization/rqt_topic/pull/51
```suggestion
  <node pkg="rqt_topic" exec="rqt_topic">
    <param name="use_sim_time" value="true"/>
    <param name="qos_overrides./air_pressure.subscription.reliability" value="best_effort" />
  </node>
```

Reply:
https://docs.ros.org/en/jazzy/Concepts/Intermediate/About-Quality-of-Service-Settings.html#qos-compatibilities was very useful in learning which QoS settings are compatible between publishers and subscribers.